tests/SassSpecTest.php

```diff
@@ -172,6 +171,14 @@ public function testTests($name, $input, $output)
             return;
         }
 
+        if (strpos($name, 'libsass-closed-issues/issue_1801/import-cycle') ||
+            strpos($name, 'libsass-todo-issues/issue_1801/simple-import-loop') ||
+            strpos($name, 'libsass-todo-issues/issue_221262') ||
+            strpos($name, 'libsass-todo-issues/issue_221292')
```

Comment:
this is moved here, because `issue_221292` is not an error test.

Also, I mark them as skipped all the time rather than putting them in the exclusion list, as we don't actually run them. This is similar to how I handle module specs in #155